packages/mux-player/src/template.ts

```diff
@@ -80,6 +80,8 @@ export const content = (props: MuxTemplateProps) => html`
     default-hidden-captions="${props.defaultHiddenCaptions}"
     forward-seek-offset="${props.forwardSeekOffset}"
     backward-seek-offset="${props.backwardSeekOffset}"
+    playbackrates="${props.playbackRates ?? false}"
+    hide-duration="${props.hideDuration ?? false}"
```

Comment:
note: This was another case of attributes vs. props from `getProps` stepping on toes, where, even for a boolean, I have to handle this with nullish coalescing.